planck/PLKClojureScriptEngine.m

```diff
@@ -552,6 +558,66 @@ -(void)startInitializationWithSrcPaths:(NSArray*)srcPaths outPath:(NSString*)out
                                          argList:@"path"
                                        inContext:self.context];
         
+
+        [ABYUtils installGlobalFunctionWithBlock:
+         ^JSValueRef(JSContextRef ctx, size_t argc, const JSValueRef argv[]) {
+             
+             if (argc == 1 && JSValueGetType (ctx, argv[0]) == kJSTypeString) {
+
+                 NSString* path = NSStringFromJSValueRef(ctx, argv[0]);
+
+                 NSDictionary* result = [[NSFileManager defaultManager]attributesOfItemAtPath:path error:nil];
```

Comment:
If an error occurs (maybe an incorrect path was passed, or the user can't read the attributes), you could decide to return nil, or maybe grab the localized description from the error, pass back a special map with `{:error "the description}`, or even make the API throw. Just ideas.